model/event/event_finder.go

```diff
@@ -8,6 +8,31 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+var embeddedResourceTypeKey = DataKey + "." + resourceTypeKey
+
+// UnprocessedEvents returns a query to fetch all unprocessed events
+func UnprocessedEvents() db.Q {
+	return db.Query(bson.M{
```

Comment:
I think it probably makes more sense to just return queries in the form of bson.M (or similar rather than as a db.Q

Reply:
done